ipyparallel/client/client.py

```diff
@@ -1042,6 +1042,11 @@ def _dispatch_iopub(self, msg):
             # ignore status messages if they aren't mine
             return
 
+        # Run any callback functions
+        msg_future = self._futures[msg_id]
+        for callback in msg_future.io_stream_callbacks:
```

Comment:
Let's call them iopub callbacks, so we have room to support other iopub messages in the future. Then the callback is responsible for checking message type.

Reply:
For sure, done.